docs/reference/settings.rst

```diff
@@ -826,6 +826,18 @@ Customise the behaviour of rich text fields. By default, ``RichTextField`` and `
 
 If a ``'default'`` editor is not specified, rich text fields that do not specify an ``editor`` argument will use the Draftail editor with the default feature set enabled.
 
+``WAGTAILADMIN_EXTERNAL_LINK_CONVERSION``
```

Comment:
As in a link by link option, or a field by field option? This setting is the result of the discussion above: it's nicer to convert links behind the scenes without asking the user every time if it is the right option, but the off switch is included just in case the site as a whole is using some sort of odd workflow that requires leaving internal links as external. If there's any ambiguity - the url doesn't exactly match - the user should get a confirmation screen anyway, so in most cases it will be case by case by default.

I suppose we could add a case by case "don't convert external -> internal" here on the modal, but it seems to add more complication for a scenario in which people really want their links to be external, but none of us can really come up with a very convincing motivation for that scenario so far...